src/perform_step/low_storage_rk_perform_step.jl

```diff
@@ -68,8 +68,8 @@ end
         end
         integrator.stats.nf += 1
     end
-    stage_limiter!(u, integrator, p, t + dt)
-    step_limiter!(u, integrator, p, t + dt)
+    stage_limiter!(u, integrator, p, t + dt) 
+    step_limiter!(u, integrator, p, t + dt) # I think step_limiter is in the wrong place here! -cwittens
```

Comment:
It`s called after the last update of `u` in line 67, isn't it? Why do you think it's wrong here?

Reply:
that's looks fine?